var/spack/repos/builtin/packages/armpl-gcc/package.py

```diff
@@ -177,20 +181,26 @@
 
 
 def get_os(ver):
-    spack_os = spack.platforms.host().default_os
+    platform = spack.platforms.host()
+    if platform.name == "darwin":
+        return "darwin"
```

Comment:
The name of this function is kind of misleading now. `darwin` is the platform, not the OS.